<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>565</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>565</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>565</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#5E686E;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelDeducerePersoane">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>293</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF8900;font-family:Times New Roman;
    font-weight: bold;
font-size:15px;
qproperty-alignment: AlignCenter;</string>
    </property>
    <property name="text">
     <string>DEDUCERE PERSOANE(NR. PERSOANE)</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF8900;
text-align: rightl;
font-size:23px;
font-family:Times New Roman;</string>
    </property>
    <property name="text">
     <string>CALCULATOR SALARIU 2018</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="spinNumarPersoane">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>293</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF8900;font-family:Times New Roman;
    font-weight: bold;
font-size:15px;
qproperty-alignment: AlignCenter;
border:0.5px solid black;</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#FF8900;</string>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>138</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="pushButtonCalculeaza">
     <property name="styleSheet">
      <string notr="true">color:#FF8900;font-family:Times New Roman;
    font-weight: bold;
font-size:15px;</string>
     </property>
     <property name="text">
      <string>CALCULEAZA</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>375</width>
      <height>48</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="labelSalariuNet">
         <property name="styleSheet">
          <string notr="true">color:#FF8900;font-family:Times New Roman;
    font-weight: bold;
font-size:15px;
qproperty-alignment: AlignCenter;</string>
         </property>
         <property name="text">
          <string>INTRODUCETI SALARIUL BRUT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QDoubleSpinBox" name="salariuInsert">
         <property name="styleSheet">
          <string notr="true">color:#FF8900;font-family:Times New Roman;
    font-weight: bold;
font-size:15px;
qproperty-alignment: AlignCenter;
border:0.5px solid black;</string>
         </property>
         <property name="maximum">
          <double>999999999999999.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>pushButtonCalculeaza</zorder>
   <zorder>labelSalariuNet</zorder>
   <zorder>labelDeducerePersoane</zorder>
   <zorder>title</zorder>
   <zorder>spinNumarPersoane</zorder>
   <zorder>salariuInsert</zorder>
   <zorder>tableWidget</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>verticalSpacer_2</zorder>
   <zorder>verticalSpacer_3</zorder>
   <zorder>splitter_2</zorder>
   <zorder></zorder>
   <zorder>splitter</zorder>
   <zorder>salariuInsert</zorder>
   <zorder>labelSalariuNet</zorder>
   <zorder>labelDeducerePersoane</zorder>
   <zorder>spinNumarPersoane</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
